docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -3215,118 +3215,83 @@ split_words('Once upon a time');
             url="https://github.com/hpcc-systems/HPCC-Platform/tree/master/testing/regress/ecl">https://github.com/hpcc-systems/HPCC-Platform/tree/master/testing/regress/ecl/</ulink></para>
           </listitem>
         </orderedlist>
-      </sect2>
-
-      <sect2 id="R" role="brk">
-        <title>R</title>
-
-        <para>To enable R support within The HPCC
-        Systems<superscript>®</superscript> Platform:</para>
-
-        <orderedlist>
-          <listitem>
-            <para>Install R-core distribution of your choice:</para>
-
-            <para><emphasis role="bold">RPM-based systems:</emphasis></para>
-
-            <para>On an RPM-based system (CentOS/Red Hat) use <emphasis
-            role="bold">R-core</emphasis> and <emphasis
-            role="bold">R-core-devel</emphasis></para>
 
-            <para><emphasis role="bold">Debian-based
-            systems:</emphasis></para>
+        <sect3>
+          <title>Python Scope Options</title>
 
-            <para>For a Debian-based (Ubuntu) system use <emphasis
-            role="bold">r-base-core</emphasis>.</para>
-          </listitem>
+          <para><emphasis role="bold">GLOBALSCOPE</emphasis> - This option
+          allows independent EMBED attributes to share globals with each other
+          if they specify the same name for the GLOBALSCOPE parameter.</para>
 
-          <listitem>
-            <para>Install the <emphasis role="bold">Rcpp</emphasis> and the
-            <emphasis role="bold">RInside</emphasis> libraries.</para>
+          <para><emphasis role="bold">PERSIST</emphasis> - This option
+          controls how long such a shared global scope will persist and
+          exactly how far it will be shared.</para>
 
-            <para>You will need both Rcpp and RInside libraries in order for
-            the R-embedding to work properly. The library installation
-            packages are named with the version number appended so you should
-            search for Rcpp_ and RInside_ to obtain the latest version for
-            your system.</para>
+          <para>The GLOBALSCOPE The value passed to GLOBALSCOPE can be any
```

Comment:
Extra words in this sentence?
The GLOBALSCOPE The value passed to GLOBALSCOPE can be any...
s/b 
The value passed to GLOBALSCOPE can be any